### DISCUSSION TITLE
Collapse button is tiny

### DISCUSSION THREAD

**Discussion by vikke1234** [piroor/treestyletab#3558] [Q&A] [answered]
Hi,

I've now installed treestyle tab on a linux machine with firefox. The button for collapsing the tree is very difficult to click and can effectively only be clicked on the yellow area in the image attached. 
![clickable area](https://github.com/piroor/treestyletab/assets/12417140/dcdf96c0-8fb4-4a94-a9dd-8f07c2d51c80)
Would anyone have an idea how I could fix it, it seems that other themes can't even be clicked on to collapse it.

**Comment by irvinm** [comment, marked as answer]
I'm not sure I understand your comment about only being able to click on the yellow part to collapse the tree.  You should be able to click on any portion of the "twisty" to collapse/expand the tree.

As for the size, I just make mine a little bigger with some TST CSS:

```
tab-item tab-twisty {
  scale: 1.25;
}
```

![image](https://github.com/piroor/treestyletab/assets/979729/fba6b865-600d-469b-ac6c-2231ab47f348)

**Reply by vikke1234** [reply]
Yeah, that was my issue though, that the twisty wasn't clickable, but thanks! The CSS made it much better.
